build/vsts_build.yaml

```diff
@@ -200,7 +200,7 @@ phases:
       solution: "build\\build.proj"
       msbuildVersion: "15.0"
       configuration: "$(BuildConfiguration)"
-      msbuildArguments: "/t:BuildVS15VSIX /p:BuildRTM=$(BuildRTM) /p:ExcludeTestProjects=$(BuildRTM) /p:IsCIBuild=true"
+      msbuildArguments: "/t:BuildVS15VSIX /p:BuildRTM=$(BuildRTM) /p:ExcludeTestProjects=$(BuildRTM) /p:IsCIBuild=true /p:BuildNumber=$(BuildNumber)"
```

Comment:
why?

Reply:
this shouldn't be needed anymore.

Reply:
yeah this isn't needed now, earlier I was trying different solution